src/SidebarProvider.ts

```diff
@@ -27,7 +33,23 @@ export class SidebarProvider implements vscode.WebviewViewProvider {
           if (!data.value) {
             return;
           }
-          vscode.window.showInformationMessage(`Hi ${data.value}, welcome to the interview!`);
+          this._socket.emit(USER_READY, {
+            sessionId: this._sessionId,
+            userName: data.value,
+          });
+          if (vscode.workspace.workspaceFolders) {
+            const workspace = vscode.workspace.workspaceFolders?.[0];
+            // Todo: fetch from our API ? what file to open as the initial starting point
+            const filePath = path.join(workspace.uri?.fsPath, "a1.txt");
```

Comment:
Replace this hardcode file name to automatically open a file after user enter their name, but I imagine this would be where we want to call our API to see what kind of files to open?

Reply:
@etchao can confirm, but I think we were just going to have a convention for this, such as always the readme maybe?

Reply:
Do you have to open a file, or can you just open a directory? The code will be cloned into `/home/workspace/data/repo`